src/store/localStorage.js

```diff
@@ -0,0 +1,25 @@
+const STATE_KEY = 'state';
+
+export const loadState = () => {
+    try{
+        const serializedState = localStorage.getItem(STATE_KEY);
+        if(serializedState === null) {
+            return undefined;
+        }
+        console.log('State loaded.');
+        return JSON.parse(serializedState);
+    } catch(err) {
+        console.log('Error: load state from local storage failed.');
+        return undefined;
```

Comment:
For such cases it is better to create variable that you are going to return and set some default value (like `undefined` in your example). In block `try` you will set value to this variable and won't return value. Then return variable after block `catch`.